lib/routes/highwaterApi.js

```diff
@@ -20,12 +20,12 @@
 var _ = require('lodash');
 var crypto = require('crypto-js');
 var log = require('../log.js')('highwaterApi.js');
-
+var amoeba = require('amoeba')
 /*
  Http interface for group-api
  */
-module.exports = function (userApiClient, kissClient, salt) {
-
+module.exports = function (salt, eventsProducer) {
+  var message = {}
```

Comment:
This variable should be defined in every method, not globally

Reply:
ah ok. done